lib/repl.js

```diff
@@ -585,10 +585,20 @@ REPLServer.prototype.createContext = function() {
     context = global;
   } else {
     context = vm.createContext();
-    for (var i in global) context[i] = global[i];
-    context.console = new Console(this.outputStream);
-    context.global = context;
-    context.global.global = context;
```

Comment:
What does `global` in the inner context refer to now? I think it should still be `context`, right?

(The second line here also seems to have been superfluous?)